README.md

```diff
@@ -11,6 +11,10 @@ Requirements
 - [SFML 2.5](http://www.sfml-dev.org/download.php)
 - [CSFML 2.5](http://www.sfml-dev.org/download/csfml/)
 
+Usage
+=====
+You can find documentation about `rust-sfml` into the [dedicated wiki](https://github.com/jeremyletang/rust-sfml/wiki)
```

Comment:
Technically, the (API) "documentation" lives at https://docs.rs/sfml/0.14.0/sfml/ . Maybe we should also include that information, to avoid people thinking that the entire documentation is on the wiki.

Also, what do you mean by dedicated wiki? To me, the word dedicated here sounds like we made our own dedicated wiki instead of using GitHub's built-in wiki feature, which is not true.